kubejs/server_scripts/vintage_improvements/recipes.js

```diff
@@ -570,6 +570,39 @@ function registerVintageImprovementsRecipes(event) {
 		processingTime: 100
 	}).id('tfg:vi/vacuum/rubber')
 
+	// #region Vaccuming rubber wood stuff for latex
+	// This would be a lot nicer if VI allowed for tags as recipe inputs but oh well...
+	Ingredient.of("#tfg:latex_logs").itemIds.forEach(log =>
+		event.custom({
+			type: 'vintageimprovements:vacuumizing',
+			ingredients: [{ item: `${log}` }],
```

Comment:
You can change this to `ingredients: [{ tag: 'tfg:latex_logs' }]` and then you can remove the foreach